modules/conversion-center/widgets/link-in-bio.php

```diff
@@ -0,0 +1,1200 @@
+								'Spotify',
+								'SoundCloud',
+								'Vimeo',
+								'Waze',
+							],
+						],
+					],
+					[
+						'name'        => 'icon_number',
+						'label'       => esc_html__( 'Number', 'elementor' ),
+						'type'        => Controls_Manager::TEXT,
+						'dynamic'     => [
+							'active' => true,
+						],
+						'label_block' => true,
+						'placeholder' => esc_html__( 'Enter your number', 'elementor' ),
+						'condition'   => [
+							'icon_platform' => [
+								'Telephone',
+								'Telegram',
+								'WhatsApp',
+							],
+						],
+					],
+					[
+						'name'        => 'icon_mail',
+						'label'       => esc_html__( 'Mail', 'elementor' ),
+						'type'        => Controls_Manager::TEXT,
+						'placeholder' => esc_html__( 'Enter your email', 'elementor' ),
+						'dynamic'     => [
+							'active' => true,
+						],
+						'label_block' => true,
+						'condition'   => [
+							'icon_platform' => [
+								'Email',
+							],
+						],
+					],
+				],
+				'title_field'   => "<i class='{{{ icon_icon.value }}}' ></i> {{{ icon_platform }}}",
+				'prevent_empty' => true,
+				'button_text'   => esc_html__( 'Add Icon', 'elementor' ),
+			]
+		);
+
+		$this->end_controls_section();
+	}
+
+
+	private function add_style_tab(): void {
+
+		$this->start_controls_section(
+			'identity_section_style',
+			[
+				'label' => esc_html__( 'Identity', 'elementor' ),
+				'tab'   => Controls_Manager::TAB_STYLE,
+			]
+		);
+
+		$this->add_control(
+			'identity_image_size',
+			[
+				'label'          => esc_html__( 'Image Size', 'elementor' ),
+				'type'           => Controls_Manager::SLIDER,
+				'default'        => [
+					'size' => 30,
+					'unit' => '%',
+				],
+				'tablet_default' => [
+					'unit' => '%',
+				],
+				'mobile_default' => [
+					'unit' => '%',
+				],
+				'size_units'     => [ 'px', '%', 'em', 'rem', 'vw', 'custom' ],
+				'range'          => [
+					'px' => [
+						'min'  => 0,
+						'max'  => 1000,
+						'step' => 1,
+					],
+					'%'  => [
+						'min' => 0,
+						'max' => 100,
+					],
+				],
+				// TODO: uncomment and adjust attributes if necessary when content controls are available
+				// 'condition' => [
+				// 	'image_style' => 'profile',
+				// ],
+				'selectors'      => [
+					'{{WRAPPER}} e-link-in-bio__identity-image' => 'width: {{SIZE}}{{UNIT}};',
+				],
+			]
+		);
+
+		$this->add_control(
+			'identity_image_shape',
+			[
+				'label'     => esc_html__( 'Image Shape', 'elementor' ),
+				'type'      => Controls_Manager::SELECT,
+				'default'   => '50%',
+				'options'   => [
+					'50%' => esc_html__( 'Circle', 'elementor' ),
+					'0'   => esc_html__( 'Square', 'elementor' ),
+				],
+				// TODO: uncomment and adjust attributes if necessary when content controls are available
+				// 'condition' => [
+				// 	'image_style' => 'profile',
+				// ],
+				'selectors' => [
+					'{{WRAPPER}} e-link-in-bio__identity-image' => 'border-radius: {{VALUE}};',
```

Comment:
```suggestion
					'{{WRAPPER}} .e-link-in-bio__identity-image' => '--identity-image-border-radius: {{VALUE}};',
```

- I think you missed a period
- Can you please use CSS variables in all your selectors instead of regular selectors.

Reply:
@louiswol94 yes the period was a miss that is addressed in a later pr.
Thanks for the re-direct onto using CSS variables. I was considering the same but was following existing examples in documentation at this time

Reply:
@willhowat The documentation is probably outdated by now. Maybe check some of our newer widgets instead. Thanks!

Reply:
@louiswol94 this should now be aligned to use custom properties